src/lexfloatclient.ts

```diff
@@ -304,6 +304,20 @@ export class LexFloatClient {
 		}
 	}
 
+	/**
+	 * Sends the request to lease the license from the LexFloatServer for offline usage.
+	 * The maximum value of lease duration is configured in the config.yml of LexFloatServer 
+	 *
+	 * @param {number} leaseDuration seconds to lease the license for
+	 * @throws {LexFloatClientException}
```

Comment:
@muneebkq Please refer to the header file docs for lease duration.

Reply:
value of the lease duration.